### ISSUE TITLE
Bug: corrupts pug variable names that are assigned to bound Vue props

### ISSUE THREAD

**Issue by bitencode** [prettier/plugin-pug#250]
### Plugin Version

v1.15.3 & 1.16.0-0

### Prettier Version

v2.3.1

### Which frameworks are affected?

- [ ] none
- [X] vue
- [ ] angular
- [ ] svelte

### Node Version

v12.19.1

### Which operating systems have you used?

- [ ] Linux
- [X] macOS
- [ ] Windows

### Prettier config

```json
{
  arrowParens: 'always',
  bracketSpacing: true,
  jsxBracketSameLine: false,
  printWidth: 100,
  proseWrap: 'preserve',
  semi: true,
  singleQuote: true,
  tabWidth: 2,
  trailingComma: 'es5',
  useTabs: false,
  pugAttributeSeparator: 'none',
  pugClassNotation: 'literal',
  pugEmptyAttributes: 'none',
  pugExplicitDiv: true,
  pugFramework: 'vue',
  pugIdNotation: 'literal',
  pugSingleFileComponentIndentation: true,
  pugSingleQuote: false,
  pugSortAttributes: 'asc',
  pugSortAttributesBeginning: ['^:?is', '^v-for', '^v-', '^:?id$', '^:?key$', '^:?ref$'],
  pugSortAttributesEnd: ['^:', '^v-on$', '^@', '^v-html$', '^v-text$'],
  pugWrapAttributesThreshold: 1,
}
```

### Input

```pug
- const circleSize = 72
- const circleWidth = 7
- v-progress-circular(
  :color="getProgressColor(currentCapacity)"
  :size=circleSize
  :value="capacityPercentage"
  :width=circleWidth
) {{ currentCapacity }}/{{ maxCapacity }}
```

### Output or Error

```pug
- const circleSize = 72
- const circleWidth = 7
v-progress-circular(
  :color="getProgressColor(currentCapacity)"
  :size="ircleSiz"
  :value="capacityPercentage"
  :width="ircleWidt"
) {{ currentCapacity }}/{{ maxCapacity }}
```

### Expected Output

```pug
- const circleSize = 72
- const circleWidth = 7
v-progress-circular(
  :color="getProgressColor(currentCapacity)"
  :size=circleSize
  :value="capacityPercentage"
  :width=circleWidth
) {{ currentCapacity }}/{{ maxCapacity }}
```

### Additional Context

~I expect if someone forgot to wrap an attribute value in quotes that `lint --fix` or `yarn prettier --write src/**/*.vue` would wrap the value correctly in the configured quote style.  Instead it replaces the first and last character of the attribute value with the quote.  Example diff after running `yarn prettier --write src/**/*.vue`:~
If a pug variable is used as the value of a bound Vue prop, it should not be wrapped in quotes:
<img width="643" alt="GitKraken" src="https://user-images.githubusercontent.com/787250/124516060-bd41be00-dd95-11eb-8e6c-7259f3ef835e.png">
Identical results on version 1.15.3 and 1.16.0-0.

**Comment by bitencode** [comment]
Updated title and description based on new understanding of what the actual problem is.

**Comment by Shinigami92** [comment]
Released in beta [v1.16.0-1](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.16.0-1)

**Comment by bitencode** [comment]
@Shinigami92 I tested `v1.16.0-1` on our code - it changed nearly 400 files (mainly prop order, stack/wrap attributes, and some line length).  So, did exactly what I'm looking for 👍 ❤️.  Outside of the #251 issue, I don't see any problems - I'm manually fixing the `v-for` problem in our code because I don't find any Vue documentation that says it's valid anyway.

Thanks! 👍
